src/app/shared/code-editor/code-editor.component.ts

```diff
@@ -57,13 +59,44 @@ export class CodeEditorComponent implements AfterViewInit {
       fontSize: '12pt',
     });
 
-    this.editor.getSession().on('change', () => {
-      this.contentChange.emit(this.editor.getValue());
-    });
-
     // Set content if possible
-    if (this.editorContent) {
-      this.editor.setValue(this.editorContent, -1);
+    const setContent = (content: string, cursorPos = 0): void => {
+      this.editorContent = content;
+
+      this.editor.getSession().on('change', () => {
+        this.contentChange.emit(this.editor.getValue());
+      });
+
+      if (this.editorContent) {
+        this.editor.setValue(this.editorContent, cursorPos);
+      }
+
+      this.editor.focus();
+    };
+
+    let sampleCodeUrl = '';
+
+    // Load helloworld files by default when editing empty CWL/WDL/NF files
```

Comment:
https://github.com/dockstore/dockstore/blob/develop/dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java#L61 are they ways we describe Nextflow